docs/designers-developers/developers/tutorials/create-block/readme.md

```diff
@@ -0,0 +1,31 @@
+
+# Create Block Tutorial
+
+The goal of this tutorial is to get you started creating your first block for the WordPress Block Editor. We will create a simple block that allows the user to type a message and styles it.
+
+The tutorial includes setting up your development environment, tools, and getting comfortable with the new development model. If you are already comfortable, try the quick start below, otherwise step through whatever part of the tutorial you need.
+
+## Quick Start
+
+The `@wordpress/create-block` package exists to create the necessary block scaffolding to get you started. See [create-block package documentation](https://www.npmjs.com/package/@wordpress/create-block) for additional features. This quick start assumes you have a development environment with node installed, and a WordPress site. From your plugins directory, to create your block run:
+
+```bash
+npx @wordpress/create-block starter-block
+cd starter-block
+npm run build
+```
+
+The above commands create a new directory called `starter-block`, installs the necessary files and builds the block plugin.
```

Comment:
Should we include a note that CLI ask you a few questions to help customize the initial setup of the block?

Reply:
👍

Reply:
Added.